app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,31 @@
+class NotesController < ApplicationController
+
+  def index
+    @notes = Note.all
+    render json: @notes
+  end
+
+  def create
+    @note = Note.new(note_params)
+    if @note.save
+      tag_names
+      render json: @note, status: :created, location: @post
+    else
+      render json: @note.errors.full_messages, status: :bad_request
+    end
+  end
+
+private
+
+  def note_params
+    params.permit(:title, :body, :tag)
+  end
+
+  def tag_names
+    tags = params[:tags].split(", ")
```

Comment:
Better to split on a bare comma and strip whitespace. There's no test covering this, but what if a user submits `{tags: 'tag1,tag2,tag3'}`